devops/kubernetes/charts/origin/templates/graphql.deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "graphql.fullname" . }}
+  labels:
+    app: {{ template "graphql.fullname" . }}
+    app.kubernetes.io/name: origin
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: backend
+    app.kubernetes.io/part-of: origin-graphql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "graphql.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "graphql.fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: origin-graphql
+        image: "{{ .Values.containerRegistry }}/{{ .Release.Namespace }}/{{ .Values.graphqlImage }}:{{ .Values.graphqlImageTag }}"
+        imagePullPolicy: Always
```

Comment:
Might need a definition of the port here, i.e.:

`ports:
    - containerPort: 4001
`

It'll still work without it but its good practice.